### PR TITLE
DHFPROD-5067: Revert all changes to entity types

### PR DESCRIPTION
**Pull request by akshaysonvane**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx [hunk 1]
```diff
@@ -30,6 +30,87 @@ describe('Entity Modeling', () => {
       cy.resetTestUser();
   });
 
+  it('can add new properties to existing Entity, and revert the entity', () => {
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx:33]: Could you add this test in front of one of the current tests? For e2e testing, I think it's better to have longer tests that simulate real user interactions.

File: marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx [hunk 2]
```diff
@@ -30,6 +30,87 @@ describe('Entity Modeling', () => {
...
+    confirmationModal.getYesButton(ConfirmationType.RevertEntity).click();
+    confirmationModal.getRevertEntityText().should('exist');
+    confirmationModal.getRevertEntityText().should('not.exist');
+
+    propertyTable.getMultipleIcon('orderID').should('not.exist');
+    propertyTable.getPiiIcon('orderID').should('not.exist');
+    propertyTable.getWildcardIcon('orderID').should('not.exist');
+  });
+
+  it('can add new properties to existing Entities, and revert all entities', () => {
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx:63]: These actions can be in front of the save all entity test.

File: marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx (hunk 1 above)
Comment (akshaysonvane) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx]: Updated tests.

File: marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx (hunk 2 above)
Comment (akshaysonvane) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx]: Updated tests.

**Comment by akshaysonvane** [comment]
Resolved conflicts.

**Comment by akshaysonvane** [comment]
Fixed typo

**Pull request merged**